src/types/Survey.ts

```diff
@@ -0,0 +1,15 @@
+export type Survey = {
+  id: string;
+  title?: Record<string, string | undefined>;
+  questionTitle: Record<string, string | undefined>;
+  description?: Record<string, string | undefined>;
+  dates: {
+    dateStart: string;
+    dateEnd: string;
+  };
```

Comment:
is string for dates correct? as it is not used here elsewhere i assume this could be changed later (and is probably already, right?)

Reply:
on the other hand, what the api returns is string, jo

<img width="202" alt="Bildschirmfoto 2021-06-18 um 16 37 47" src="https://user-images.githubusercontent.com/1942953/122577836-86f50680-d053-11eb-96f4-7907e15e9f2c.png">

Reply:
string is correct as this is the type we get from the query, e.g. `2020-12-20`

there is a change from `dates` -> `date` in a future PR because the field name was actually incorrect

Reply:
ah true, i see now as well